packages/container/lib/container.js

```diff
@@ -1,6 +1,6 @@
 import { dictionary, symbol, setOwner, OWNER } from 'ember-utils';
 import { ENV } from 'ember-environment';
-import { assert, deprecate, runInDebug } from 'ember-metal';
+import { assert, deprecate, runInDebug, NAME_KEY } from 'ember-metal';
```

Comment:
Can you move `NAME_KEY` (which I think is just a symbol right?) into the `ember-utils` package instead?  We are ultimately trying to make `container` package not rely on ember-metal (the `assert`, `deprecate`, `runInDebug` functionality was the only remaining bit).